config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-	post '/signup', to: 'users#signup'
+  post '/signup', to: 'users#signup'
+  post '/auth/login', to: 'authentication#login'
```

Comment:
Hello @wrotich, any reason we are using a different controller for login, I mean since we already have the users controller that handles for signup already?

Reply:
Yeah, This makes sense, let me look into it

Reply:
Done